### ISSUE TITLE
[Bug]: No UI Indication for Focus on Switch Component During Keyboard Navigation

### ISSUE THREAD

**Issue by shivamchawla-msft** [microsoft/fluentui#33546]
### Component

Switch

### Package version

3.0.0-beta.70

### @microsoft/fast-element version

2.0.0

### Environment

```shell
System:
    OS: Windows 11 10.0.22631
    CPU: (16) x64 AMD EPYC 7763 64-Core Processor
    Memory: 34.07 GB / 63.95 GB
  Browsers:
    Edge: Chromium (130.0.2849.52)
    Internet Explorer: 11.0.22621.3527
  npmPackages:
    @fluentui/tokens: 1.0.0-alpha.20 => 1.0.0-alpha.20
    @fluentui/web-components: 3.0.0-beta.70 => 3.0.0-beta.70
    @microsoft/fast-element: 2.0.0 => 2.0.0
```

### Current Behavior

There is no UI indication to the user that the focus is on switch upon keyboard navigation. 

### Expected Behavior

There should be some UI indication to the user that the focus is on switch upon keyboard navigation. 

### Reproduction

https://github.com/shivamchawla-msft/fluent-ui-switch-focus-repro/tree/main

### Steps to reproduce

1. clone the project
2. pnpm install
3. pnpm run
4. open the  URL in the terminal with the port. 
5. navigate to the switch using tab button of keyboard.
6. Invoke it via space to make sure focus is on switch only
7. Notice there is no indication to the user that keyboard focus is on switch.  

### Are you reporting an Accessibility issue?

yes

### Suggested severity

Medium - Has workaround

### Are you willing to submit a PR to fix?

yes

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by chrisdholt** [comment]
Thanks for this issue - it looks like you're willing to contribute a PR to address this, that would be much appreciated! @davatron5000 as FYI

**Comment by davatron5000** [comment]
I looked into this and there is a working `:focus-visible` state on Switch. 

Screenshots from https://web-components.fluentui.dev/?path=/docs/components-switch--docs

Standalone Switch:
![Image](https://github.com/user-attachments/assets/ac931e8f-dc05-4e1d-bdc2-fdcd5b006229)

Inside Field Component:
![Image](https://github.com/user-attachments/assets/92e8b42d-843d-4fee-bd66-fbb297f83af3)

I took a look at the repro and noticed that `<fluent-field>` wasn't being imported at the top of `index.ts`. 

```diff
import '@fluentui/web-components/label.js';
import '@fluentui/web-components/switch.js';
+ import '@fluentui/web-components/field.js';
```

Adding that line gets the following result when starting the server.

![Image](https://github.com/user-attachments/assets/baeb4548-c115-4e42-8108-bcec76e907a5)

Closing this issue now per investigation. Please re-open if further action is needed.

**Comment by shivamchawla-msft** [comment]
Thank you for pointing it out.
